<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmulationSettingsWidget</class>
 <widget class="QWidget" name="EmulationSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="speedGroup">
     <property name="title">
      <string>Speed Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Slow-Motion Speed:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="fastForwardSpeed"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="slowMotionSpeed"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Normal Speed:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Fast-Forward Speed:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="normalSpeed"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="systemSettingsGroup">
     <property name="title">
      <string>System Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1">
       <widget class="QComboBox" name="eeCycleRate">
        <item>
         <property name="text">
          <string>50% (Underclock)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60% (Underclock)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>75% (Underclock)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100% (Normal Speed)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>130% (Overclock)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>180% (Overclock)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>300% (Overclock)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="eeCycleSkipping">
        <item>
         <property name="text">
          <string>Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mild Underclock</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Moderate Underclock</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Maximum Underclock</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="systemSettingsLayout">
        <item row="1" column="0">
         <widget class="QCheckBox" name="cheats">
          <property name="text">
           <string>Enable Cheats</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="MTVU">
          <property name="text">
           <string>Enable Multithreaded VU1 (MTVU)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="hostFilesystem">
          <property name="text">
           <string>Enable Host Filesystem</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="fastCDVD">
          <property name="text">
           <string>Enable Fast CDVD</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="precacheCDVD">
          <property name="text">
           <string>Enable CDVD Precaching</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="threadPinning">
          <property name="text">
           <string>Enable Thread Pinning</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>EE Cycle Rate:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>EE Cycle Skipping:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pacingGroup">
     <property name="title">
      <string>Frame Pacing / Latency Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QSpinBox" name="maxFrameLatency">
        <property name="suffix">
         <string extracomment="This string will appear next to the amount of frames selected, in a dropdown box."> frames</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Maximum Frame Latency:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QGridLayout" name="basicCheckboxGridLayout">
        <item row="0" column="1">
         <widget class="QCheckBox" name="syncToHostRefreshRate">
          <property name="text">
           <string>Sync to Host Refresh Rate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="vsync">
          <property name="text">
           <string>Vertical Sync (VSync)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="optimalFramePacing">
          <property name="text">
           <string>Optimal Frame Pacing</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="useVSyncForTiming">
          <property name="text">
           <string>Use Host VSync Timing</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="skipPresentingDuplicateFrames">
          <property name="text">
           <string>Skip Presenting Duplicate Frames</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="rtcGroup">
     <property name="title">
      <string>Real-Time Clock</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutRTC">
      <item row="0" column="0">
       <widget class="QCheckBox" name="manuallySetRealTimeClock">
        <property name="text">
         <string>Manually Set Real-Time Clock</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateTimeEdit" name="rtcDateTime"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
